robot/Cumulus/resources/RecurringDonationsPageObject.py

```diff
@@ -18,10 +18,16 @@ class RDListingPage(BaseNPSPPage, ListingPage):
     @capture_screenshot_on_error
     def wait_for_rd2_modal(self):
         """Based on the button name (Cancel)  or (Save) on the modal footer, selects and clicks on the respective button"""
-        self.builtin.sleep(2,"Wait Needed for now to wait for the new modal")
+        self.builtin.sleep(3,"Wait Needed for now to wait for the new modal")
```

Comment:
Any way an implicit wait could be used here? (Wait For Element to be Present, etc)

Reply:
Changed it with a better way of waiting

Reply:
👍